Overview/Capabilities/permgroups.html

```diff
@@ -1,12 +1,12 @@
-<?xml version="1.0" encoding="utf-8"?>
-
-<mixer template="gw.tmpl">
-<mixertitle>Permutation and Matrix Groups</mixertitle>
+---
+title: Permutation and Matrix Groups
+layout: default
+---
 
 <p>
   For an overview of computational methods for permutation groups see
   the book [<a href="/Doc/references.html#Se03">Se03</a>] of
-  <mixer person="Akos Seress" data="name"/>. Historically, these methods
+  Akos Seress (1958-2013). Historically, these methods
```

Comment:
I think he is written **Á**kos Seress.
Do we want to be consistent with writing years behind names?

Reply:
Weirdly, in the database (and on the old page) it was without the accent and for some reason the years were part of his name in there too. Happy to take it or leave it.

Reply:
Raised globally in issue #160 . Thanks Paula!